### ISSUE TITLE
Remove 'Show on map' hover from the service tree

### ISSUE THREAD

**Issue by kallelongjuhani** [City-of-Helsinki/servicemap#434]
It is redundant with the 'Show' buttons.

**Comment by tituomin** [comment]
@kallelongjuhani is this still a good idea in light of the recent user testing in Iiris? Although the user in question didn't actually notice the hovers, but could maybe have...

**Comment by kallelongjuhani** [comment]
Yeah, maybe better leave it as is. When and if we make the Show-button more
explicit, then we can reconsider removing the hover label.

Y. t.

Kalle
—
+358 (0)50 357 2769
kalle.jarvenpaa@gmail.com
Tehtaankatu 25 B 46
00150 Helsinki

2016-04-27 15:54 GMT+03:00 Timo Tuominen <notifications@github.com>:
